grammar.js

```diff
@@ -178,6 +181,46 @@ module.exports = grammar({
         'end',
       ),
 
+    if: $ => seq(
+      'if',
+      field('condition', $._expression),
+      choice($._terminator, field('consequence', $.then)),
```

Comment:
Do we need to save the `then` to a field?

Reply:
Ah I guess you're using it to store the body of the if, would recommend not putting all 3 of these together in one field if possible